ruby_exercises/mythical-creatures/lib/centaur.rb

```diff
@@ -0,0 +1,39 @@
+class Centaur
+  attr_reader :name,
+              :breed,
+              :shoot,
+              :run,
+              :cranky,
+              :standing
+      
+
+  def initialize(name, breed = 'Palomino')
```

Comment:
I wouldn't put a default breed here. Usually the tests will say something like, a Centaur has a default breed of `<insert breed name>` and not pass the breed parameter through. In all of the tests, 2 parameters are passed in, so it's safe to assume breed is a require parameter.